### PR TITLE
Reset cache

### PR DESCRIPTION
**Pull request by shreesh1993**
- Added script to generate meta.json with package.json version number
- Added component to check if there is need to refresh new browser for new updates

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -97,15 +97,17 @@
...
     "test": "react-scripts test",
     "test:coverage": "react-scripts test --coverage --watchAll",
     "cy:run": "cypress run",
     "eject": "react-scripts eject",
     "format": "prettier \"**/*.{ts,tsx,css,graphql}\" --write",
     "floweditor": "cp -r node_modules/@glific/flow-editor/build/static public && rm -rf public/sitestatic && cp -r node_modules/@glific/flow-editor/build/sitestatic public",
-    "extract-translations": "i18next --fail-on-warnings"
+    "extract-translations": "i18next --fail-on-warnings",
+    "generate-build-version": "node generate-build-version",
+    "prebuild": "npm run generate-build-version"
```
Comment (kurund) [inline package.json:110]: please use yarn.

File: src/App.tsx [hunk 2]
```diff
@@ -12,12 +12,15 @@ import { UnauthenticatedRoute } from './route/UnauthenticatedRoute/Unauthenticat
 import { AuthenticatedRoute } from './route/AuthenticatedRoute/AuthenticatedRoute';
 import { Logout } from './containers/Auth/Logout/Logout';
 import { Loading } from './components/UI/Layout/Loading/Loading';
+import handleCache from './HandleCache';
```
Comment (kurund) [inline src/App.tsx:15]: This should be implemented as a service.

**Review by kurund — CHANGES_REQUESTED** [review]
Not sure if this PR is relevant and can be closed infavor of https://github.com/glific/glific-frontend/pull/1320. Good to check with @DigneshGujarathi

**Comment by shreesh1993** [comment]
Closing this for now..

**Pull request closed without merging**